internal/iam/account.go

```diff
@@ -58,7 +59,7 @@ func (a *authAccount) VetForWrite(ctx context.Context, r db.Reader, opType db.Op
 }
 
 func (a *authAccount) validScopeTypes() []scope.Type {
-	return []scope.Type{scope.Global, scope.Org}
+	return scope.AllowedIn(resource.Account)
 }
```

Comment:
Do we still need the old `validScopeTypes` functions, or should we just replace the call sites with this new function instead?

Reply:
We still need this function because `validScopeTypes` is a method of the `ResourceWithScope` interface (used in `validateScopeForWrite`)